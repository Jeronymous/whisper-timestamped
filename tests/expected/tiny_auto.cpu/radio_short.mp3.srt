1
00:00:00,060 --> 00:00:06,100
What are you telling me, guys?

2
00:00:06,760 --> 00:00:07,620
What are you telling me, dude?

3
00:00:07,620 --> 00:00:08,400
What are you telling me, dude?

4
00:00:11,200 --> 00:00:11,860
What are you telling me, dude?

5
00:00:11,860 --> 00:00:13,140
What are you telling me, guys?

6
00:00:20,240 --> 00:00:21,480
What are you telling me, guys?

7
00:00:22,080 --> 00:00:22,820
What are you telling me, dude?

8
00:00:22,820 --> 00:00:23,680
What are you telling me, dude?

9
00:00:26,040 --> 00:00:28,800
What are you telling me, dude?

10
00:00:30,920 --> 00:00:32,480
What are you telling me, dude?

11
00:00:33,480 --> 00:00:34,220
What are you telling me, dude?

12
00:00:34,220 --> 00:00:35,640
What are you telling me, dude?

13
00:00:36,240 --> 00:00:38,480
What are you telling me, dude?

14
00:00:38,480 --> 00:00:39,080
What are you telling me, dude?

15
00:00:39,520 --> 00:00:41,960
What are you telling me, dude?

16
00:00:41,960 --> 00:00:42,880
What are you telling me, dude?

17
00:00:44,940 --> 00:00:46,480
What are you telling me, dude?

18
00:00:46,480 --> 00:00:47,080
What are you telling me, dude?

19
00:00:47,520 --> 00:00:48,520
What are you telling me, dude?

20
00:00:49,520 --> 00:00:51,420
What are you telling me, dude?

21
00:00:51,980 --> 00:00:54,250
What are you telling me, dude?

22
00:00:54,250 --> 00:00:55,160
What are you telling me, dude?

23
00:00:56,680 --> 00:00:58,060
What are you telling me, dude?

24
00:00:58,060 --> 00:00:59,160
What are you telling me, dude?

25
00:00:59,520 --> 00:01:01,160
What are you telling me, dude?

26
00:01:01,520 --> 00:01:02,960
What are you telling me, dude?

27
00:01:03,520 --> 00:01:05,570
What are you telling me, dude?

28
00:01:05,570 --> 00:01:07,120
What are you telling me, dude?

29
00:01:07,520 --> 00:01:09,440
What are you telling me, dude?

30
00:01:10,860 --> 00:01:12,200
What are you telling me, dude?

31
00:01:12,200 --> 00:01:12,920
What are you telling me, dude?

32
00:01:14,040 --> 00:01:14,840
What are you telling me, dude?

33
00:01:15,520 --> 00:01:18,120
What are you telling me, dude?

34
00:01:18,120 --> 00:01:19,220
What are you telling me, dude?

35
00:01:20,220 --> 00:01:21,280
What are you telling me, dude?

36
00:01:22,040 --> 00:01:23,100
What are you telling me, dude?

37
00:01:24,340 --> 00:01:25,440
What are you telling me, dude?

38
00:01:25,520 --> 00:01:27,980
What are you telling me, dude?

39
00:01:27,980 --> 00:01:30,210
What are you telling me, dude?

40
00:01:30,210 --> 00:01:30,810
What are you telling me, dude?

41
00:01:31,520 --> 00:01:32,320
What are you telling me, dude?

42
00:01:33,540 --> 00:01:34,680
What are you telling me, dude?

43
00:01:36,900 --> 00:01:38,540
What are you telling me, dude?

44
00:01:38,540 --> 00:01:39,140
What are you telling me, dude?

45
00:01:40,960 --> 00:01:41,560
What are you telling me, dude?

46
00:01:41,560 --> 00:01:43,320
What are you telling me, dude?

47
00:01:45,060 --> 00:01:45,660
What are you telling me, dude?

48
00:01:46,840 --> 00:01:48,000
What are you telling me, dude?

49
00:01:48,000 --> 00:01:49,760
What are you telling me, dude?

50
00:01:50,040 --> 00:01:50,880
What are you telling me, dude?

51
00:01:52,140 --> 00:01:53,700
What are you telling me, dude?

52
00:01:53,900 --> 00:01:55,000
What are you telling me, dude?

53
00:01:55,520 --> 00:01:57,740
What are you telling me, dude?

54
00:01:58,180 --> 00:01:59,240
What are you telling me, dude?

55
00:01:59,520 --> 00:02:00,500
What are you telling me, dude?

56
00:02:01,520 --> 00:02:03,600
What are you telling me, dude?

57
00:02:03,600 --> 00:02:05,320
What are you telling me, dude?

58
00:02:05,520 --> 00:02:07,640
What are you telling me, dude?

59
00:02:09,260 --> 00:02:09,860
What are you telling me, dude?

60
00:02:09,860 --> 00:02:11,860
What are you telling me, dude?

61
00:02:11,860 --> 00:02:13,180
What are you telling me, dude?

62
00:02:13,760 --> 00:02:15,140
What are you telling me, dude?

63
00:02:15,520 --> 00:02:17,840
What are you telling me, dude?

