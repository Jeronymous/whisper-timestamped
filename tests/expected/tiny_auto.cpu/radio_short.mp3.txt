What are you telling me, guys?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, guys?
What are you telling me, guys?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
What are you telling me, dude?
